scripts/repl.js

```diff
@@ -204,6 +204,29 @@
     evt.stopPropagation();
   }
 
+  function envItemToString(item) {
+    return item.name + ' ' + JSON.stringify(item.targets);
+  }
+
+  function onEnvBuild(node, opts, envResult) {
```

Comment:
Also, seems like we depend on https://github.com/yavorsky/babel-preset-env-standalone, so maybe move it to babel/babel-preset-env-standalone? I can ask him 🙂.